src/pages/FrenchFamilyBenefitsExample.res

```diff
@@ -71,28 +81,46 @@ type allocationsFamilialesOutput =
   | Result(float)
   | Error(React.element)
 
+let formatResidenceKindInput = (residenceKind: string) => {
+  if residenceKind == `Métropole` {
+    `Metropole`
+  } else if residenceKind == `La Réunion` {
+    `LaReunion`
+  } else if residenceKind == `Saint Barthélemy` {
+    `SaintBarthelemy`
+  } else if residenceKind == `Saint Martin` {
+    `SaintMartin`
+  } else if residenceKind == `Saint Pierre et Miquelon` {
+    `SaintPierreEtMiquelon`
+  } else {
+    residenceKind
+  }
+}
+
 let validateInput = (input: allocationsFamilialesInput) => {
-  switch (input.currentDate, input.numChildren, input.income, input.residence) {
-  | (Some(currentDate), Some(_numChildren), Some(income), Some(residence)) =>
+  switch (input.currentDate, input.numChildren, input.income, input.residenceKind) {
+  | (Some(currentDate), Some(_numChildren), Some(income), Some(residenceKind)) =>
     let childrenValidated = input.children->Belt.Array.map(child => {
       switch (child.birthDate, child.monthlyIncome) {
       | (Some(birthDate), Some(monthlyIncome)) =>
         Some({
-          dateNaissance: birthDate,
-          id: child.id,
-          remunerationMensuelle: monthlyIncome,
-          priseEnCharge: {
+          dDateDeNaissance: birthDate,
+          dIdentifiant: child.id,
+          dRemunerationMensuelle: monthlyIncome->Belt.Int.toFloat,
+          dPriseEnCharge: {
             switch child.priseEnCharge {
-            | None => "Effective et permanente"
-            | Some(s) => s
+            | None => {kind: "EffectiveEtPermanente", payload: None}
+            | Some(kind) => {kind: kind, payload: None}
             }
           },
-          aDejaOuvertDroitAuxAllocationsFamiliales: {
+          dADejaOuvertDroitAuxAllocationsFamiliales: {
             switch child.aDejaOuvertDroitAllocationsFamiliales {
             | None | Some(false) => false
             | Some(true) => true
             }
           },
+          // TODO: add input for this field
+          dBeneficieTitrePersonnelAidePersonnelleLogement: false,
```

Comment:
@denismerigoux Should an entry be added for this field?

Reply:
I added this TODO because the field wasn't present, but I can add it anyway :sweat_smile: